.github/workflows/react-tests-dispatcher.yml

```diff
@@ -0,0 +1,30 @@
+name: REACT Tests Dispatcher
+
+on:
+  push:
+    branches: [ "main", "7.*", "8.*" ]
+    paths:
+      - 'rules/**/*.toml'
+
+jobs:
+  dispatch:
+    name: Run REACT tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger REACT workflow
+        env:
+          REACT_WORKFLOW_ID: "detection-rules-test"
```

Comment:
Just double checking, does this workflow ID need to match https://github.com/elastic/react/pull/250/files#diff-fe9be20922a1703b2eb04ba2c32719c3f226b808113f76c68d44873a492bad45 ?

Reply:
good one! It needs to match a filename and I'm missing `s` there. Thank you!